### PR TITLE
gudp_ntohs and gudp_htons

### PR DESCRIPTION
**Pull request by Lucashsmello**
Hi, i need these two functions (gudp_ntohs and gudp_htons) for a custom gimx code. 
If you are interested of what is this custom gimx code, its just some additional network functions for controlling GIMX (related to https://github.com/matlo/GIMX/pull/618).

### CODE REVIEW

File: include/gudp.h [hunk 1]
```diff
@@ -70,12 +70,30 @@ uint32_t gudp_ntohl(uint32_t netlong);
...
+ * \param hostlong  the unsigned integer to convert
  *
  * \return the converted unsigned integer
  */
 uint32_t gudp_htonl(uint32_t hostlong);
 
+/*
+ * \brief Convert an unsigned short integer from host byte order to network byte order.
+ *
+ * \param hostshort  the unsigned integer to convert
```
Comment (matlo) [inline include/gudp.h]: unsigned integer => unsigned short

File: include/gudp.h [hunk 2]
```diff
@@ -70,12 +70,30 @@ uint32_t gudp_ntohl(uint32_t netlong);
...
+ *
+ * \return the converted unsigned short integer
+ */
+uint16_t gudp_htons(uint16_t hostshort);
+
+/*
+ * \brief Convert an unsigned short integer from network byte order to host byte order.
+ *
+ * \param netshort  the unsigned integer to convert
```
Comment (matlo) [inline include/gudp.h]: unsigned integer => unsigned short

**Review by matlo — CHANGES_REQUESTED** [review]

File: include/gudp.h (hunk 1 above)
Comment (matlo) [reply include/gudp.h]: or maybe unsigned short integer

File: include/gudp.h (hunk 2 above)
Comment (matlo) [reply include/gudp.h]: or maybe unsigned short integer

File: include/gudp.h (hunk 1 above)
Comment (Lucashsmello) [reply include/gudp.h]: Personally, i prefer unsigned short integer.

**Pull request merged**